docs/multicluster/antctl.md

```diff
@@ -26,5 +26,59 @@ antctl mc get clusterset [NAME] [-n NAMESPACE] [-o json|yaml] [-A]
 antctl mc get resourceimport [NAME] [-n NAMESPACE] [-o json|yaml] [-A]
 antctl mc get resourceexport [NAME] [-n NAMESPACE] [-clusterid CLUSTERID] [-o json|yaml] [-A]
 ```
-
 To see the usage examples of these commands, you may also run `antctl mc get [subcommand] --help`.
+
+## antctl mc create
+
+`antctl mc create` command can create access-token and other resources like ClusterSet, ClusterClaims for 
```

Comment:
I think we need to clarify we support `antctl mc get` since v1.6 and support other commands in v1.7.